.vscode/launch.json

```diff
@@ -1,17 +1,22 @@
 {
   "version": "0.2.0",
   "configurations": [
-    {
-      "type": "PowerShell",
-      "request": "launch",
-      "name": "ps: Interactive Session",
-      "cwd": "${workspaceRoot}"
+  {
```

Comment:
Since we already have this launch script defined under the Mvc directory we can revert this change.

Reply:
Reverted!